MOSH_README.md

```diff
@@ -0,0 +1,64 @@
+# Using `mosh` with Fluent Terminal
+
+`mosh` can be used in Fluent Terminal in two ways:
+
+ 1. Through GUI, by turning ***Use mosh*** option in *SSH Connection* dialog;
+ 2. As a custom command, in *Quick Launch* dialog.
+
+The first way, through GUI is very simple and self-descriptive, and it will not be explained here. Using `mosh` as a custom command is more complicated, but also offers much more options. The simplest `mosh` command you can enter is:
+
+    mosh user@host
+
+This will initiate a `mosh` session with default options (i.e. port range will default to `60000:61000`, etc.).
```

Comment:
This will initiate a `mosh` session with default options (i.e. port range will default to `60000:61000`).

Reply:
Changed in https://github.com/felixse/FluentTerminal/pull/572